content/fr/distributed-systems.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Système distribué
+status: Feedback Appreciated
+category: concept
+tags: ["architecture"]
+---
+
+## Ce que c'est
+
+Un système distribué est un ensemble de composants autonomes connectés via un réseau, et qui apparaît à ses utilisateurs comme un seul système cohérent.
+La plupart du temps appelés [nœuds](/nodes/), ces composants peuvent aussi bien être des machines physiques (des ordinateurs ou des téléphones portables par exemple) que des processus logiciels.
+Les nœuds sont programmés pour atteindre un objectif commun et échangent des messages via le réseau afin de collaborer.
+
+## Problème auquel il répond
+
+De nos jours, de nombreuses applications modernes sont si conséquentes de par leur taille qu'elles nécessiteraient de superordinateurs afin de pouvoir fonctionner.
```

Comment:
I'm not sure to be honest. Take [the example from the Larousse page on the `nécessiter` conjugation](https://www.larousse.fr/dictionnaires/francais/n%C3%A9cessiter/54012):

> Ce travail nécessite de longues recherches.

While not being able to point you out to the French rule for that precise case right now, I'm quite confident that `de superordinateurs` is correct; WDYT?

Reply:
Yes it would work if you had an adjective before "superordinateur" like "nécessiterait de splendides superordinateurs", but without it we must keep "des" (after some search it seems linked to the rules applied for "adjectifs épithètes")

Reply:
Indeed, thanks for looking that up before I did! I got tricked by the "super" part being close to an "adjectif épithète" :stuck_out_tongue: 

```suggestion
De nos jours, de nombreuses applications modernes sont si conséquentes de par leur taille qu'elles nécessiteraient des superordinateurs afin de pouvoir fonctionner.
```